client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.jsx

```diff
@@ -155,52 +176,33 @@ const DataFilesBreadcrumbs = ({
       }
     }, '');
 
+  const fullPath = paths.slice(-1);
+  const currentDirectory = pathComps.slice(-1);
+
   return (
-    <div className={`breadcrumbs ${className}`}>
-      {scheme === 'projects' && (
-        <>
-          <RootProjectsLink
-            api={api}
-            section={section}
-            operation={operation}
-            label="Shared Workspaces"
-          />{' '}
-          {system && `/ `}
-        </>
```

Comment:
Previously we had the "Shared Workspaces" root in the breadcrumb path. Do we want to include that with this feature as well?
<img width="481" alt="Screenshot 2023-10-18 at 12 16 08 PM" src="https://github.com/TACC/Core-Portal/assets/20326896/e615d4f6-f538-4eb2-b01e-e50fb7c71bcd">

Reply:
<img width="1260" alt="SharedProjects" src="https://github.com/TACC/Core-Portal/assets/50084480/16fcdaa0-f5a5-43df-9550-1bef68954b16">
What do you think of this? I'll fix the folder icon alignment, but do you like this idea for Shared Workspaces?

Reply:
Looks good!